### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by niranjanakoni**

### CODE REVIEW

**Review by sirmv — APPROVED** [review]

**Review by sirmv — CHANGES_REQUESTED** [review]
need improvement

**Comment by niranjanakoni** [comment]
changes are made

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# GHA
\ No newline at end of file
+# GHA
+adkjasdl
+asdsadsad
+
+sdsaadasd
+
+sdasasd
```
Comment (sirmv) [inline README.md]:
```suggestion

sdsaadasd

sdasasd
```

remove these lines

Comment (niranjanakoni) [reply README.md]: done

**Review by sirmv — APPROVED** [review]

**Review by sirmv — APPROVED** [review]

**Review by sirmv — APPROVED** [review]

**Pull request merged**